main.tf

```diff
@@ -0,0 +1,55 @@
+provider "aws" {
+  alias = "glue"
+}
```

Comment:
Is this due to a copy and paste action, of is this really needed?

Reply:
do you mean the alias or the provider itself? I see in all MCAF modules a provider is used with alias

Reply:
A provider (both with and without an alias) is only needed when you need to create the resources with a custom provider. If you can use the already initialized parent provider, this is almost always a better choice.

So in this case I would remove the provider block until (if ever) we need to be able to pass a specific provider.

Reply:
thanks for the explanation